tests/unit/CrowdingComponent.spec.js

```diff
@@ -0,0 +1,36 @@
+import { shallowMount } from "@vue/test-utils";
+import CrowdingComponent from "@/components/CrowdingComponent.vue";
+
+describe("CrowdingComponent", () => {
+  describe("Methods", () => {
+    test("searchLocation should return the search location and set haveSearched to true", () => {
+      const data = {
+        haveSearched: false,
+        selectedTime: 0,
+        selectedLocation: "391 Amsterdam Ave",
+        time: null
+      };
+
+      const wrapper = shallowMount(CrowdingComponent, {
+        propsData: data
+      });
+      expect(data.selectedLocation).toMatchInlineSnapshot(
+        `"391 Amsterdam Ave"`
+      );
+    });
+  });
+
+  test("returns correctly", () => {
+    const wrapper = shallowMount(CrowdingComponent, {
+      props: { busyResults: "", hasSearched: false }
```

Comment:
I think this is supposed to be `propsData`, I'm not sure if `props` does anything.

Reply:
Maybe it's working because the default values for the props are similar to these.

Reply:
Ah, yeah, I think you're right. It should be propsData